conf/messages.cy

```diff
@@ -201,6 +201,16 @@ form.export-questions.vessel-details.dateOfDepartureOptional=Dyddiad cyrraedd (d
 form.export-questions.vessel-details.timeOfDepartureOptional=Amser cyrraedd amcangyfrifedig (dewisol)
 form.export-questions.vessel-details.timeOfDepartureOptional.hint=Er enghraifft, 14:30
 form.export-questions.vessel-details.dateOfDeparture.hint=Er enghraifft, {0}
+
```

Comment:
This Welsh here is correct but I've just noticed there are a few bits where we're using arrival instead of departure. 

The following lines we need to substitute cyrraedd (arrival) with **ymadael** (departure)

196 
form.export-questions.vessel-details.dateOfDeparture=Dyddiad cyrraedd

197
form.export-questions.vessel-details.timeOfDeparture=Amser cyrraedd amcangyfrifedig

200
form.export-questions.vessel-details.dateOfDepartureOptional=Dyddiad cyrraedd (dewisol)

201
form.export-questions.vessel-details.timeOfDepartureOptional=Amser cyrraedd amcangyfrifedig (dewisol)

334
summary.vessel-details.dateOfDeparture=Dyddiad cyrraedd

336
summary.vessel-details.timeOfDeparture=Amser cyrraedd amcangyfrifedig